tests/test_data/thepioneerwoman.com/thepioneerwoman_3.json

```
@@ -0,0 +1,55 @@
+
+{
+  "author": "Erica Kastner",
+  "canonical_url": "https://www.thepioneerwoman.com/food-cooking/recipes/a98271/how-to-make-cream-cheese-whipped-cream/",
+  "category": "dessert,main dish",
+  "cook_time": null,
+  "cuisine": "",
+  "description": "I\u2019ll bet just reading the title of this post made you say, \u201cCream cheese whipped cream?",
+  "host": "thepioneerwoman.com",
+  "image": "https://hips.hearstapps.com/thepioneerwoman/wp-content/uploads/2018/06/cream-cheese-whipped-cream-01.jpg?crop=0.6668334167083542xw:1xh;center,top&resize=1200:*",
+  "ingredient_groups": [
+    {
+      "ingredients": [
+        "8 oz. weight Cream Cheese, Softened",
+        "2 tbsp. Pure Maple Syrup, Or More To Taste",
+        "2 c. Heavy Whipping Cream",
+        "1 tsp. Lemon Zest",
+        "1/4 tsp. Pure Vanilla Extract"
+      ],
+      "purpose": null
+    }
+  ],
+  "ingredients": [
+    "8 oz. weight Cream Cheese, Softened",
+    "2 tbsp. Pure Maple Syrup, Or More To Taste",
+    "2 c. Heavy Whipping Cream",
+    "1 tsp. Lemon Zest",
+    "1/4 tsp. Pure Vanilla Extract"
+  ],
+  "instructions": "Place the cream cheese and maple syrup in a large bowl. Beat until completely smooth. Scrape down the sides of the bowl.Pour 1/4 of the cream into the cream cheese mixture. Beat until completely smooth. Scrape down the sides of the bowl. Repeat 3 more times for a total of 4 additions.Add the lemon zest and vanilla and beat until smooth. Store in the refrigerator until ready to use. If you want the cream cheese whipped cream to be soft, make sure to take out of the refrigerator for about 15 minutes and give it a good whisk before serving.Makes enough to frost a layer cake made with two 6-inch layers.",
+  "instructions_list": [
+    "Place the cream cheese and maple syrup in a large bowl. Beat until completely smooth. Scrape down the sides of the bowl.Pour 1/4 of the cream into the cream cheese mixture. Beat until completely smooth. Scrape down the sides of the bowl. Repeat 3 more times for a total of 4 additions.Add the lemon zest and vanilla and beat until smooth. Store in the refrigerator until ready to use. If you want the cream cheese whipped cream to be soft, make sure to take out of the refrigerator for about 15 minutes and give it a good whisk before serving.Makes enough to frost a layer cake made with two 6-inch layers."
```

```suggestion
    "Place the cream cheese and maple syrup in a large bowl. Beat until completely smooth. Scrape down the sides of the bowl.",
    "Pour 1/4 of the cream into the cream cheese mixture. Beat until completely smooth. Scrape down the sides of the bowl. Repeat 3 more times for a total of 4 additions.",
    "Add the lemon zest and vanilla and beat until smooth. Store in the refrigerator until ready to use. If you want the cream cheese whipped cream to be soft, make sure to take out of the refrigerator for about 15 minutes and give it a good whisk before serving.",
    "Makes enough to frost a layer cake made with two 6-inch layers."
```

Change related to the adjustment of the instructions gathering, this list will now be broken up by the `\n`